### DISCUSSION TITLE
A (possibly) simpler way to ask for help

### DISCUSSION THREAD

**Discussion by PandaFoss** [archlinux/archinstall#226] [Ideas]
I don't know if this is an idea per se, but rather a suggestion. I find the [Termbin](https://termbin.com/) project interesting, and I think it could be used by users when asking for help.
We know that the `archinstall` log file is located in `/var/log/archinstall/install.log`. And we also know that the `gnu-netcat` package is installed by default in the Arch Linux ISO, so the user could run the following:

```bash
nc termbin.com 9999 < /var/log/archinstall/install.log
```

In a few seconds the terminal will return a link in the form `termbin.com/XXXX` (where `XXXX` is a random alphanumeric code). The user should then just share that link to receive help. It is even possible to add an alias or similar, to avoid having to remember the command (although this is not very complex either).

**Comment by Torxed** [comment]
Thought I ended up on a parked domain at first. But it's a netcat reciever that acts as a pastebin?
I think we discussed pastebins at some point, but I can't remember where.
But I'm not sure I'm comfortable suggesting this via the interface.. Never used termbin before.

I do like making things easier tho, and currently the only two options people seam to have is to SSH in/out and grab it or screenshot.
I'll circle back here once I'm done with all the PR's and some issues! Thanks for the tip!
